src/kibana/components/visualize/visualize.html

```diff
@@ -2,7 +2,7 @@
   class="text-center visualize-error visualize-chart">
   <div class="item top"></div>
   <div class="item">
-    <h2><i class="fa fa-meh-o"></i></h2>
+    <h2><i aria-hidden="true" class="fa fa-meh-o"></i></h2>
```

Comment:
Will screen readers have a problem here with the "empty" h2, or is just because the screen readers try and read the ASCII key?